### PR TITLE
Add 0.164 release notes

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.164.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+---------------
+
+* Fix correctness issue for queries that perform ``DISTINCT`` and ``LIMIT`` on the results of a ``JOIN``.
+* Fix correctness issue when casting between maps where the key or value is the ``REAL`` type.
+* Fix correctness issue in :func:`min_by` and :func:`max_by` when nulls are present in the comparison column.
+* Fail queries when ``FILTER`` clause is specified for scalar functions.
+* Fix planning failure for certain correlated subqueries that contain aggregations.
+* Fix planning failure when arguments to selective aggregates are derived from other selective aggregates.
+* Fix boolean expression optimization bug that can cause long planning times, planning failures and coordinator instability.
+* Fix query failure when ``TRY`` or lambda with the exact same body is repeated in an expression.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.164.rst]: lambda -> lambda expression

File: presto-docs/src/main/sphinx/release/release-0.164.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Web UI Changes
+--------------
+
+* Add resource group to query detail page.
+
+Hive Changes
+------------
+
+* Fix handling of ORC files containing extremely large metadata.
+* Improve query scheduling when reading bucketed tables.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.164.rst]: Is this a hive or general change? Is it an improvement in performance?

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.164.rst]: @haozhun please clarify

Comment (haozhun) [inline presto-docs/src/main/sphinx/release/release-0.164.rst]: @martint and I agreed on: Improve performance for queries that read bucketed tables by optimizing scheduling.

File: presto-docs/src/main/sphinx/release/release-0.164.rst [hunk 3]
```diff
@@ -27,4 +28,5 @@ Hive Changes
 ------------
 
 * Fix handling of ORC files containing extremely large metadata.
+* Fix creating views in file based metastore.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.164.rst]: What's the failure (i.e., what's broken? under what conditions?)

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.164.rst]: It always fails. How about "Fix failure when creating ..."

**Pull request closed without merging**